netlify.toml

```diff
@@ -15,12 +15,12 @@
   force = true
 [[redirects]]
   from = "/api"
-  to = "/api/blockchain"
+  to = "/"
   status = 301
   force = true
 [[redirects]]
   from = "/api/blockchain/*"
-  to = "/blockchain/api/:splat"
+  to = "/"
```

Comment:
`/blockchain/api/` also needs a redirect.

Reply:
updated to redirect to `/oracles/oracle-data`